qna/DE-archive-data-storage.md

```diff
@@ -22,31 +22,33 @@ lang-en, market-de, data-archive, bundle-carefree, PosCreator, PosDealer, PosOpe
 
 ### Type of data
 
-Client-related data (= name, address, contact data, UID number, orders, etc. from cash register manufacturers, cash register dealers, cash register operators)
-Configuration and status data (= TSE certificate, ClientId, status of the queue, etc.)
-Mass data (= ReceiptRequest, ReceiptResponse)
-Revision-secured client-related mass data (= .Tar-File, DSFinV-K-File, etc.)
-General customer-related bulk data (= copy of everything that is manually stored in revision-secured customer-related bulk data, invoices, contracts, etc.)
+* Client-related data (= name, address, contact data, UID number, orders, etc. from cash register manufacturers, cash register dealers, cash register operators)
+
+* Configuration and status data (= TSE certificate, ClientId, status of the queue, etc.)
+* Mass data (= ReceiptRequest, ReceiptResponse)
+
+* Revision-secured client-related mass data (= .Tar-File, DSFinV-K-File, etc.)
+* General customer-related bulk data (= copy of everything that is manually stored in revision-secured customer-related bulk data, invoices, contracts, etc.)
 
 ### Processing locations
 
-Azure Region West-Europe (a104, a105)
-Azure-Region Germany-West-Central  (a110, a111)
-Azure-Region France-Central (a107, a108)
-Equinix-Datacenter Amsterdam (nl18) (https://www.equinix.com/locations/europe-colocation/netherlands-colocation/amsterdam-data-centers/)
-RRZ-Datacenter Raaba / Graz (at11) (https://www.rrz.co.at/)
+* Azure Region West-Europe (a104, a105)
+* Azure-Region Germany-West-Central  (a110, a111)
+* Azure-Region France-Central (a107, a108)
+* Equinix-Datacenter Amsterdam (nl18) (https://www.equinix.com/locations/europe-colocation/netherlands-colocation/amsterdam-data-centers/)
+* RRZ-Datacenter Raaba / Graz (at11) (https://www.rrz.co.at/)
 
```

Comment:
same like in German content: Why mentioning France (e.g.) in Germany data storage?